tests/hf_hydrodata/test_gridded.py

```diff
@@ -1842,6 +1842,24 @@ def test_pf_flow_barrier():
     assert data[3, 3, 4] == 0.001
 
 
+def test_get_pfb_vegm_with_default_masking():
+    """Test that get_gridded_data() does not mask vegm files."""
+
+    bounds = [200, 200, 202, 202]
```

Comment:
Would these bounds normally lead to NaNs in the returned data?

Reply:
Hmm. Good question. I think so, but it would have been better to have a test that used 0, 0 which I'm sure would return NaN. Following up I should probably have a test for a point in the Great Lakes which should also return NaN before. Good ideas.